wd_helper.au3

```diff
@@ -1015,16 +1017,19 @@ Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand, $vParameters
 					$vResult = _WD_ExecuteScript($sSession, $sScript, __WD_JsonElement($sSelectElement), Default, $_WD_JSON_Value)
 					$iErr = @error
 
-				Case 'multiSelect', 'singleLabel' ; https://stackoverflow.com/a/1296068/5314940
+				Case 'multiSelect', 'SINGLESELECT' ; https://stackoverflow.com/a/1296068/5314940
```

Comment:
Why is this in ALL CAPS?

Reply:
please take a look on: [singleSelect capitalizastion](https://github.com/Danp2/au3WebDriver/pull/367/commits/6f2b1ce2c358831bd204e4cebaa554007c69b8f3)